lucille-core/src/test/java/com/kmwllc/lucille/stage/DropDocumentTest.java

```diff
@@ -0,0 +1,69 @@
+package com.kmwllc.lucille.stage;
+
+import static org.junit.Assert.*;
+
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import org.junit.Test;
+
+public class DropDocumentTest {
+
+  private final StageFactory factory = StageFactory.of(DropDocument.class);
+
+
+  @Test
+  public void testPublisher() {
```

Comment:
publishing happens at the beginning of the pipeline
we can't "drop" a document unless it has first been published
there are already tests in RunnerTest to make sure that dropped documents are handled properly by the lucille framework
DropDocumentsTest should focus only on making sure that the DropDocument stage does what it is supposed to do